adapters/custom-image-upload.adapter.js

```diff
@@ -0,0 +1,210 @@
+/* globals XMLHttpRequest, FormData, console */
+
+/**
+ * Upload adapter.
+ *
+ * @private
+ * @implements module:upload/filerepository~UploadAdapter
+ */
+class CustomImageUploadAdapter {
+	/**
+	 * Creates a new adapter instance.
+	 *
+	 * @param {module:upload/filerepository~FileLoader} loader
+	 * @param {path: ./adapters/customimageuploadadapter~CustomImageUploadConfig} options
+	 */
+	constructor( loader, options ) {
```

Comment:
Is this our code or is this copy pasted from somewhere? If it is copy pasted from somewhere, it's worth adding a link to the source, etc, otherwise - this code does not comply with our standards and I don't think the .prettierc file is present in repo. If this is our code - it should comply with our standards. Otherwise - plz add reference/links/etc + comments stating that

Reply:
added the links which I followed at the top of the page